code-samples/additional-info/rest-api/13/index.js

```diff
@@ -11,7 +11,7 @@ client.once('ready', () => {
 	console.log('Ready!');
 });
 
-client.on('message', async message => {
+client.on('messageCreate', async message => {
 	if (!message.content.startsWith(prefix) || message.author.bot) return;
 
 	const args = message.content.slice(prefix.length).trim().split(/ +/);
```

Comment:
The `.send(file)` and `.send(embed)` lines in this file need to be updated to their respective `MessageOptions` properties